x/tokenregistry/keeper/keeper.go

```diff
@@ -71,6 +71,21 @@ func (k keeper) GetDenom(ctx sdk.Context, denom string) types.RegistryEntry {
 	}
 }
 
+func (k keeper) GetIBCDenom(ctx sdk.Context, ibcdenom string) types.RegistryEntry {
```

Comment:
Needs a more descriptive name and comment. It get's confusing with ordinary denom / ibc usage, when this is specific to denom conversions for ibc.